home/app_manager.py

```diff
@@ -234,17 +253,64 @@ def _refresh_prereleases(self, change):
                 self.include_prereleases.value or prerelease_installed
             )
 
+    def find_unmatched_requirements(self, app_version, python_bin):
+        """Calling find_incompatibilities and get unmatched requirements"""
+        return [
+            r[1] for r in self.app._app.find_incompatibilities(app_version, python_bin)
+        ]
+
+    def find_to_be_installed_dependencies(self, app_version, python_bin=None):
+        """return a list of Dependency"""
+        if python_bin is None:
+            python_bin = sys.executable
+
+        if app_version is None:
+            return []
+
+        dependencies = []
+        unmatched_requirements = self.find_unmatched_requirements(
+            app_version, python_bin
+        )
+        packages = find_installed_packages(python_bin)
+        for requriment in unmatched_requirements:
```

Comment:
typo, needs to be fixed below as well
```suggestion
        for requirement in unmatched_requirements:
```